src/modules/TextOverlay/TextOverlay.js

```diff
@@ -22,6 +23,28 @@ module.exports = exports = function(pixels, options, url1, cb){
     ctx.fillStyle = options.color;
     ctx.font = options.size + 'px ' + options.font;
     ctx.fillText(options.text, options.x, options.y);
+    
+    var write = ctx.measureText(options.text);
+    ctx.beginPath();
+    ctx.strokeStyle = options.color;
+    ctx.lineWidth = 2;
```

Comment:
We can also provide this as an input

Reply:
Updated PR @blurry-x-face

Reply:
Updated the test for text-overlay module too